packages/manager/src/components/ColorPalette/ColorPalette.tsx

```diff
@@ -13,30 +13,29 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
   swatchWrapper: {
     display: 'flex',
-    flexDirection: 'column',
     alignItems: 'center',
-    justifyContent: 'center',
-    width: '15%',
+    marginBottom: theme.spacing(),
   },
   swatch: {
-    content: '',
-    height: 64,
-    width: 64,
-    margin: theme.spacing(2),
+    border: '1px solid #888f91',
+    borderRadius: 3,
+    height: theme.spacing(4),
+    width: theme.spacing(4),
```

Comment:
Can we make the swatches a little bit bigger?

```suggestion
    height: theme.spacing(5),
    width: theme.spacing(5),
```

Reply:
I compromised at `theme.spacing(4.5)` lol